screen/budgets/src/main/java/com/ivy/budgets/BudgetViewModel.kt

```diff
@@ -262,3 +273,12 @@ class BudgetViewModel @Inject constructor(
         }
     }
 }
+
+    fun calculateTotalRemainingBudget(
+        budgets: ImmutableList<DisplayBudget>,
+        categoryBudgetsTotal: Double
+    ): Double {
+        return categoryBudgetsTotal - budgets
+            .filter { !it.budget.categoryIdsSerialized.isNullOrBlank() }
```

Comment:
use `isNotNullOrBlank()` and also add a comment that you're filtering to only Category budgets